teps/0079-tekton-catalog-support-tiers.md

```diff
@@ -100,14 +128,460 @@ requirements, and provenance attestations can be made to meet software supply ch
 [TEP-0091: Verified Remote Resources][tep-0091] will flesh out the details of signing, while this TEP will focus on
 surfacing the verification information and building a corpus of verified resources that users can trust.
 
+## Definitions
+
+The keywords “MUST” and “MAY” are to be interpreted as described in 
+[RFC 2119][rfc2119].
+
+Terms used in this TEP are defined as follows:
+* **Catalog**: A repository that complies with the organization contract
+  defined in [TEP-0003: Tekton Catalog Organization][tep-0003-org].
+* **Resource**: Item shared in a Tekton Catalog e.g. `Task` or `Pipeline`.
+* **Tekton Catalog Maintainers**: The core group of OWNERS who can approve
+  changes in Tekton Catalogs. They are defined in [OWNERS][catalog-owners]
+  file in the existing Catalog at `tektoncd/catalog`.
+
+## Proposal
+
+### Ownership and Maintenance
+
+As previously discussed in [TEP-0003][tep-0003-upstream], we propose
+creating two support tiers, `Community` and `Official`, through 
+separate Tekton Catalogs.
+
+The Community Catalog makes it easy for contributors to share resources, 
+while the Official Catalog provides a corpus of high quality resources 
+that users can rely on.
+
+Both the Community and Official Catalogs will be:
+* Owned by the [Tekton Catalog Maintainers][catalog-owners].
+  The resources in the Community Catalog are owned by contributors,
+  but the Catalog itself is owned by the Tekton Catalog Maintainers.
```

Comment:
what does it mean by owned between cotributors and maintainers? does that mean allow to approve ? allow to merge?